docs/general/contributing/direct-donations.mdx

```diff
@@ -0,0 +1,50 @@
+---
+uid: donate-directly
+title: Donations to individual Developers
+---
+
+import Link from '@docusaurus/Link';
+import { donations } from '../../../src/data/donations';
+
+Jellyfin is free and will always be free. We do not accept any money or benefit for any form of favor or feature and no implicit or explicit entitlement should be assumed or demanded when donating to jellyfin or one of its developers.
+
+That said the many developers or moderators accept donations purely as a sign of gratitude for their volunteer work on the project or officially related.
+
+This table is not conclusive and if you are missing a specific person you want to donate to, please contact the developer directly.
+This list is also intentionally randomly ordered to avoid any appearance of favoring any one developer.
+
+<table>
+  <thead>
+    <th>Username</th>
+    <th>Role</th>
+    <th>Donation Links</th>
+  </thead>
+  <tbody>
+    {donations
+      .sort(() => Math.random() - 0.5)
+      .map((element) => (
+        <tr>
+          <td>{element.title}</td>
+          <td>
+            <ul>
+              {element.roles.map((role) => (
+                <li>{role}</li>
+              ))}
+            </ul>
+          </td>
+          <td>
+            <ul>
+              {element.links.map((link) => (
+                <li>
+                  <Link to={link.url}>{link.name}</Link>
+                </li>
+              ))}
+            </ul>
+          </td>
+        </tr>
+      ))}
+  </tbody>
+</table>
+
+The rules for inclusion into this lists are a good standing with the Jellyfin team as a whole and a long term commitment towards the project as a whole.
```

Comment:
We should ONLY include active Jellyfin team members (people contributing to official projects in the jellyfin or jellyfin-labs org and in our **team** chat).

Additionally in this list, just like any other data list, the core team has the final decision.

Reply:
agreed